controllers/rhmi/rhmi_controller.go

```diff
@@ -452,41 +452,34 @@ func (r *RHMIReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 	if installation.Status.ToVersion == version.GetVersionByType(installation.Spec.Type) && !installInProgress && !productVersionMismatchFound {
 		installation.Status.Version = version.GetVersionByType(installation.Spec.Type)
 		installation.Status.ToVersion = ""
-		metrics.SetRhmiVersions(string(installation.Status.Stage), installation.Status.Version, installation.Status.ToVersion, string(externalClusterId), installation.CreationTimestamp.Unix())
-		if rhmiv1alpha1.IsRHOAM(rhmiv1alpha1.InstallationType(installation.Spec.Type)) {
-			// remove the code for next release
-			removeOldStages(installation)
-			// end of removal
-			installation.Status.Quota = installationQuota.GetName()
-			installation.Status.ToQuota = ""
-		}
+		metrics.SetVersions(string(installation.Status.Stage), installation.Status.Version, installation.Status.ToVersion, string(externalClusterId), installation.CreationTimestamp.Unix())
+		installation.Status.Quota = installationQuota.GetName()
+		installation.Status.ToQuota = ""
+		// remove the code for next release
+		removeOldStages(installation)
```

Comment:
this can go

Reply:
This is code related to RHOAM not RHMI. There is enough change happening with the RHMI removal. The removal of this is out side the scope of this work.

Reply:
Same as above - since we are doing these massive efforts of going literally through every file and cleaning up our repo to me this needs to go as well.

Reply:
As this is not a typical PR I do not agree with code changes that are not related to the task at hand. To resolve this I have opened a second PR which has the code removal you are looking for. https://github.com/integr8ly/integreatly-operator/pull/2920

Reply:
Thanks